lightning/src/ln/functional_tests.rs

```diff
@@ -2314,14 +2316,16 @@ fn test_htlc_on_chain_success() {
 		_ => panic!("Unexpected event"),
 	};
 	macro_rules! check_tx_local_broadcast {
-		($node: expr, $htlc_offered: expr, $commitment_tx: expr, $chan_tx: expr) => { {
-			// ChannelManager : 3 (commitment tx, 2*HTLC-Timeout tx), ChannelMonitor : 2 (timeout tx) * 2 (block-rescan)
+		($node: expr, $htlc_offered: expr, $commitment_tx: expr, $chan_tx: expr, $len: expr) => { {
+			// ChannelManager : 3 (commitment tx, 2*HTLC-Timeout tx), ChannelMonitor : 2 (timeout tx)
```

Comment:
This comment is similarly out of date (can you explain why one case has 7 and one 5) and can you just use the $htlc_offered flag to check for 7 vs 5 instead of adding a new parameter?

Reply:
There is diff between node 0 and node 1 because in one case the commitment tx is local and in the other case that's remote so same explanation than comment above (local HTLC-timeout are still duplicated)